x-pack/plugins/synthetics/public/apps/synthetics/components/settings/private_locations/view_location_monitors.tsx

```diff
@@ -35,7 +35,7 @@ export const ViewLocationMonitors = ({
     <EuiPopover button={button} isOpen={isPopoverOpen} closePopover={closePopover}>
       <FormattedMessage
         id="xpack.synthetics.monitorManagement.viewMonitors"
-        defaultMessage='Location "{name}" is used in {count, number} {count, plural,one {monitor} other {monitors}}.'
+        defaultMessage="{name} is configured with {count, number} {count, plural,one {monitor} other {monitors}}."
```

Comment:
```suggestion
        defaultMessage="{name} is used in {count, number} {count, plural,one {monitor} other {monitors}}."
```
Thanks! Few things here:

- I updated the wording, I think it sounds more natural than what we had initially written in the parent issue.
- In the screenshots in the PR desc, I see a "create" button underneath, wasn't it initially a "View monitors" button?
- There seems to be some logic to distinguish between singular and plural, but in the screenshot I see "0" with then "monitors" in its plural form.

Reply:
> I updated the wording, I think it sounds more natural than what we had initially written in the parent issue.

++ I'll commit right from here.

> In the screenshots in the PR desc, I see a "create" button underneath, wasn't it initially a "View monitors" button?

This is how it looks in `main` right now, we can change it if you like.

<img width="341" alt="image" src="https://github.com/elastic/kibana/assets/18429259/84494721-d83e-4d15-a1b8-5cd021bb4e56">


> There seems to be some logic to distinguish between singular and plural, but in the screenshot I see "0" with then "monitors" in its plural form.

Should it not be in the plural form for 0 monitors?

<img width="354" alt="image" src="https://github.com/elastic/kibana/assets/18429259/eab5d257-7aa2-499c-b6d0-50be2fff6198">

Reply:
With 0 monitor the ideal logic would be to negate the entire sentence `test isn't used in any monitors yet. Create monitor` but I'm fine with the current if that is too much of a hassle ^^ 

As for View monitors vs. Create: It makes sense to me to have "Create" when the location is not used in any monitor. But when it is, it would also make sense to list the monitors the location is currently used in (hence using "View monitors"). I maybe missed that logic change but it's fine if that's a product decision, I just wasn't aware :)

Reply:
> With 0 monitor the ideal logic would be to negate the entire sentence 

Ah I see. We're at FF today so I'd like to get this merged ASAP. I'll make a follow-up PR to try and add that in too, but if it doesn't make it at least we fixed the rest of the copy and we can get it in the next version.

> As for View monitors vs. Create: It makes sense to me to have "Create" when the location is not used in any monitor. But when it is, it would also make sense to list the monitors the location is currently used in (hence using "View monitors"). I maybe missed that logic change but it's fine if that's a product decision, I just wasn't aware :)

I'll do the same here and try to come back around again and get another PR in.